torch/csrc/jit/passes/quantization.cpp

```diff
@@ -81,18 +83,19 @@ void addQuantDeQuantNodes(Value* v) {
 
   // Attach inputs to quant and dequant nodes
   quant->addInput(v);
-  // Default Quant Params <Scale:1.0, ZeroPoint:0>
-  insertQuantNodeParams(quant, std::make_tuple(1.0, 0));
+
+  insertQuantNodeParams(quant, std::get<1>(qparam),
+    std::get<2>(qparam));
   dequant->addInput(quant->output());
 }
 
 // Insert Quant-Dequant node pair for specific input to node n
-void addQuantDeQuantNodesForInput(Value* v, Node* n) {
+void addQuantDeQuantNodesForInput(Value* v, Node* n,
+  std::tuple<std::string, float, int>& qparam) {
```

Comment:
You just moved away from using tuple in the function above - maybe flatten this one as well then?